tooling/template-base-essentials/components/RenderLinks.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const RenderLinks = (props: {
+  links: { [s: string]: unknown } | ArrayLike<unknown> | null;
```

Comment:
Think we can work on the type def here. Since the `links` prop receives an array of objects w/ `href` and `label`, we should be able to define a `type` for use here. Don't think we need `null` either.

Reply:
Have now added the following declaration in Render Links props: `links: Array<{ href: string; label: number }>;`